h2o-core/src/main/java/water/api/RapidsHandler.java

```diff
@@ -109,4 +111,21 @@ public InitIDV3 endSession(int version, InitIDV3 p) {
     }
     return p;
   }
+
+  public static class StartSession4 extends Handler2<InputSchemaV4, SessionIdV4> {
```

Comment:
We have a naming convention that classes ending with `V3` indicate rest-api-level-3 schemas, and classes ending with `V4` are rest-api-level-4 schemas. This class is not a schema, so calling it `StartSessionV4` would be confusing.
On a brighter side, this name really doesn't matter because I will be moving this class out of `RapidsHandler` and into the `CascadeHandler`, where it will be simply called `StartSession`.